web/components/TextBlock/TextAndImage/TextAndImage.tsx

```diff
@@ -31,8 +33,18 @@ export const TextAndImage = ({
         <div className={styles.text}>
           {title && (
             <hgroup>
-              <Heading type="h2">{title}</Heading>
-              {tagline && <h3 className={styles.tagline}>{tagline}</h3>}
+              <Heading type="h2" id={`heading-h2-${_key}`}>
+                {title}
+              </Heading>
+              {tagline && (
+                <Heading
+                  type="h3"
+                  id={`heading-h3-${_key}`}
+                  className={styles.tagline}
+                >
```

Comment:
This should not be using `Heading`.

Also, as above I doubt that it's useful to the editors to have an ID here anyway, since it's just a "tagline" and not the heading of a separate sub-section.

Reply:
692572d